submissions/Oleksii_Drohachov/HTML_CSS_PopUp/style.css

```diff
@@ -0,0 +1,187 @@
+  }
+  
+  body {
+    background-color: rgba(250, 235, 215, 0.662);
+  }
+  
+  ::-webkit-scrollbar,
+  ::-webkit-scrollbar-thumb {
+    width: 10px;
+    border-radius: 5px;
+    border: 1px solid rgb(138, 0, 252);
+    background-color: rgba(177, 138, 248, 0.4);
+  }
+  
+  img {
+    width: 100%;
+    height: 100%;
+  }
+  .nav__list {
+    background-color: rgba(177, 138, 248, 0.4);
+    height: 50px;
+    display: flex;
+    list-style-type: none;
+    justify-content: flex-end;
+    border-radius: 10px;
+    align-items: center;
+  }
+  
+  .nav__item,
+  .nav__item__menu {
+    display: block;
+    width: 100px;
+    height: 50px;
+    text-align: center;
+    align-items: center;
+    padding-top: 11px;
+    font-size: 17px;
+  }
+  
+  .nav_link {
+    text-decoration: none;
+    color: black;
+  }
+  
+  .nav__item:focus,
+  .nav__item__menu:focus {
+    border: 2px solid black;
+  }
+  
+  .nav__item__menu {
+    width: 50px;
+    padding-right: 15px;
+  }
+  
+  .nav__item:hover {
+    border: 2px solid grey;
+    border-radius: 5px;
+    transform: scale(1.1);
+  }
+  
+  .logo__git {
+    display: block;
+    width: 40px;
+    height: 40px;
+    border: 1px solid black;
+    border-radius: 50%;
+    margin: 0 auto;
+  }
+  
+  .popup-img {
+    display: block;
+    width: 40px;
+    height: 40px;
+    margin: 0 auto;
+  }
+  
+  .first,
+  .second {
+    position: absolute;
+    -webkit-appearance: none;
+  }
+  
+  .first {
+    background-image: url(icons/popup-button.png);
+    background-size: 100%;
+    height: 32px;
+    width: 32px;
+  }
+  
+  .second {
+    height: 25px;
+    width: 150px;
+    position: absolute;
+    transform: translate(95px, 365px);
+  }
+  
+  .first:focus,
+  .second:focus {
+    border: 1px solid black;
+  }
+  
+  .first:hover,
+  .second:hover {
+    border: 2px solid grey;
+    border-radius: 5px;
+  }
+  
+  .first:hover {
+    transform: scale(1.3);
+  }
+  
+  .first_label:hover {
+    cursor: pointer;
+  }
+  
+  .first:checked + .main__wrapper {
+    display: flex;
+    width: 360px;
+  }
+  
+  .main__wrapper {
+    display: none;
+    flex-direction: column;
+    position: relative;
+    transform: translate(-210px, 50px);
+    width: 350px;
+    background-color: rgba(177, 138, 248, 0.163);
+    border: 2px solid rgb(91, 10, 241);
+    border-radius: 10px;
+    overflow-y: scroll;
+    height: 410px;
+  }
+  
+  .menu__list__main {
+    display: grid;
+    grid-template-rows: repeat(3, 110px);
+    grid-template-columns: repeat(3, 100px);
+    gap: 15px 15px;
+    list-style-type: none;
+    padding: 0 0 0 0;
+  }
+  
+  .menu__item__main {
+    box-sizing: border-box;
+  }
+  
+  .menu__item__main:hover {
+    border: 2px solid grey;
+    border-radius: 10px;
+  }
+  
+  .item_box {
+    display: flex;
+    flex-direction: column;
+    width: 100px;
+    height: 110px;
+    font-size: 14px;
+    justify-content: center;
+    align-content: flex-start;
+  }
+  
+  .item_box > div > img {
```

Comment:
Avoid styling by HTML tag. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)
Also, avoid using too specific CSS selectors. [Read more](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)